release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -364,6 +364,85 @@ This dashboard provides API server metrics, such as:
 
 // TODO: Link to troubleshooting using this dashboard section once it is available
 
+[id="ocp-4-7-monitoring-alerting-rule-changes"]
+==== Alerting rule changes
+
+{product-title} 4.7 includes the following alerting rule changes:
+
+.Alerting rule changes
+[%collapsible]
+====
+* The `AlertmanagerClusterCrashlooping` alert is added. The critical alert provides notification if at least half of the Alertmanager instances in a cluster are crashlooping.
+* The `AlertmanagerClusterDown` alert is added. The critical alert provides notification if at least half of the Alertmanager instances in a cluster are down.
+* The `AlertmanagerClusterFailedToSendAlerts` alert is added. The critical alert provides notification if all Alertmanager instances in a cluster failed to send notifications.
+* The `AlertmanagerFailedToSendAlerts` alert is added. The warning alert provides notification if an Alertmanager instance failed to send notifications.
+* The `etcdBackendQuotaLowSpace` alert is added. The critical alert provides notification if the database size of an etcd cluster exceeds the defined quota on an etcd instance.
+* The `etcdExcessiveDatabaseGrowth` alert is added. The warning alert provides notification if there is an observed surge in etcd writes that caused a 50% increase in database size on an etcd instance over a four-hour period.
+* The `etcdHighFsyncDurations` alert is added. The critical alert provides notification if the 99th percentile fsync durations of an etcd cluster are too high.
```

Comment:
Doesn't look like fsync has been marked up in the rest of the repo yet, but... maybe `fsync`?

Reply:
etcdHighFsyncDurations is added to 4.7
```
            "name": "etcdHighFsyncDurations",
            "query": "histogram_quantile(0.99, rate(etcd_disk_wal_fsync_duration_seconds_bucket{job=~\".*etcd.*\"}[5m])) > 0.5",
```

Reply:
> Doesn't look like fsync has been marked up in the rest of the repo yet, but... maybe `fsync`?

Looks like the etcd docs has `fsync` marked up, too. 👍